core/themes/basis/template.php

```diff
@@ -38,6 +38,20 @@ function basis_preprocess_page(&$variables) {
     $variables['classes'][] = 'view-name-' . $view->name;
   }
 
+  // Get the installed version.
+  $update_version = config_get('basis.settings', 'css_update_version');
+  $update_preference = config_get('basis.settings', 'css_update');
+
+  // Process supplemental CSS versions.
+  $supplemental_css_versions = basis_supplemental_css_versions();
```

Comment:
Everywhere else we've used the term "update" (which is fine) rather than "supplemental". Let's be consistent and use "update" here too.

```suggestion
  $update_css_versions = basis_update_css_versions();
```

Reply:
Sorry to reopen this, did you decide sticking with supplemental in this situation was preferable?

Reply:
@quicksketch No, it's ok. I think I changed it, but then I made a mistake and it was reverted. 

I'll get it again.

Reply:
I think you were looking at an old version. It's fixed already.